photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala

```diff
@@ -242,6 +251,16 @@ object Params {
             "convention.")
         .foreach(x => params.offHeapIndexMapNumPartitions = x)
 
+      opt[String]("input-columns-names")
+        .text(s"A map of the input columns names, in format: response:label|offset:offset|uid:uid ... ")
+        .foreach(x =>
+          x.split("\\|")
+            .foreach { line =>
+              val Array(key, value) = line.split(":").map(_.trim)
+              params.inputColumnsNames.updated(InputColumnsNames.withName(key), value)
+            }
+        )
+
```

Comment:
I prefer the map. It's more flexible for extension, should we need to extend it. @joshvfleming What's your reason for preferring one parameter per replacement column name?